### PR TITLE
Reactive canton conformance test aginst LF 1.13

### PR DESCRIPTION
**Pull request by remyhaemmerle-da**
Now we compile the conformance test for 1.13, we can run conton
against 1.13.

follow up of #10456

CHANGELOG_BEGIN
CHANGELOG_END

### Pull Request Checklist

- [ ] Read and understand the [contribution guidelines](https://github.com/digital-asset/daml/blob/main/CONTRIBUTING.md)
- [ ] Include appropriate tests
- [ ] Set a descriptive title and thorough description
- [ ] Add a reference to the [issue this PR will solve](https://github.com/digital-asset/daml/issues), if appropriate
- [ ] Include changelog additions in one or more commit message bodies between the `CHANGELOG_BEGIN` and `CHANGELOG_END` tags
- [ ] Normal production system change, include purpose of change in description

NOTE: CI is not automatically run on non-members pull-requests for security
reasons. The reviewer will have to comment with `/AzurePipelines run` to
trigger the build.

### CODE REVIEW

File: daml-lf/language/daml-lf.bzl [hunk 1]
```diff
@@ -18,16 +18,14 @@
 
 lf_version_configuration = {
     "legacy": "1.8",
+    # TODO (MK) Drop no-exception once Canton fully supports exceptions.
```
Comment (cocreature) [inline daml-lf/language/daml-lf.bzl:21]: Have you checked that we don’t use this anywhere else? The reason why I kept it separate is that I wanted to limit the amount of stuff we have to compile & test against 1.13.

**Review by cocreature — APPROVED** [review]

File: daml-lf/language/daml-lf.bzl (hunk 1 above)
Comment (remyhaemmerle-da) [reply daml-lf/language/daml-lf.bzl:23]: Yes. 

The only place where it was really used was for the rules:
- `//daml-lf/encoder:testing-dar-lookup-scala-%s` and 
- `//daml-lf/encoder:testing-dar-lookup-lib-%s` and 
The compilation of this two rules induced a minor overhead. 

However, I double check and realized that only `//daml-lf/encoder:testing-dar-lookup-lib-latest` is used, so  deactivate in b2c4e108393ce3b631f70dcef9afca0142ad2552 the compilation for the other keywords, meaning now compile even less.

**Comment by remyhaemmerle-da** [comment]
The crux of the PR is actually in 29599b6c57e016b64fe8794c6548c3a76b459112

File: ledger/ledger-api-test-tool-on-canton/BUILD.bazel [hunk 2]
```diff
@@ -62,7 +62,7 @@ conformance_test(
         "@jdk11_nix//:bin/java",
     ],
     lf_versions = [
-        "legacy",
+        "no-exceptions",
         # FIXME: replace the previous line by the following ones once canton-test-runner supports LF 1.14
```
Comment (remyhaemmerle-da) [inline ledger/ledger-api-test-tool-on-canton/BUILD.bazel:66]: @oliverse-da The two FIXME are for you.

**Pull request merged**